Index.html

```diff
@@ -171,6 +171,79 @@ <h3 class="h3">Skills</h3>
             </ul>
         </section>
         
+        <section id="contact" class="contact">
+            <img src="./Contents/Resources/Pictures/image_geometry_1.svg" alt="" class="corner__geometry">
+            <img src="./Contents/Resources/Pictures/contact_shape_one.svg" alt="" class="contact__image__left">
+            <div class="article__form" >
+                <h2 class="h2 contact__h2">Get Started</h2>
+                <p class="contact__text">I Prefer To Speak By My Gestures</p>
+                <form action="https://formspree.io/f/" method="post" class="contact__form">
+                    <label class="label__name label" for="name">Your name <span class="red__colour">*</span></label>
+                    <input type="text" required maxlength="30" id="name" placeholder="write your name" name="name" class="name">
+                    <label class="label__email" for="email">Your email <span class="red__colour">*</span></label>
+                    <input type="email" required pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$" placeholder="write your email" id="email" name="email" class="email">
+                    <label class="label__message" for="message">Message <span class="red__colour">*</span></label>
+                    <textarea maxlength="500" required placeholder="Enter your message here..." name="message" id="message" class="message"></textarea>
+                    <span class="error" aria-live="polite"></span>
+                    <input type="submit" id="submit" value="Start Collaboration" class="buttons contact__button">
```

Comment:
- [ ] Please make sure to stick to the `Figma` design template as much as possible. Your form `inputs` label on mobile screen doesn't show. Kindly fix it so your design looks exactly like that of the `Figma`design. See below for more details: 
<table>
<tr>
<th align="middle">Figma design</th>
<th align="middle">Your design</th>
</tr>
<tr>
<td><img width="363" alt="Screenshot 2023-05-02 at 18 57 21" src="https://user-images.githubusercontent.com/107192076/235763183-5b79274d-7290-4953-8bf4-4a34c3aa150a.png"></td>
<td><img width="371" alt="Screenshot 2023-05-02 at 18 58 21" src="https://user-images.githubusercontent.com/107192076/235763251-d8cd0978-2aa5-4488-bfa7-fc01adf117f6.png"></td>
</tr>
</table>

Reply:
> * [x]  Please make sure to stick to the `Figma` design template as much as possible. Your form `inputs` label on mobile screen doesn't show. Kindly fix it so your design looks exactly like that of the `Figma`design. See below for more details:
> 
> Figma design	Your design
> <img alt="Screenshot 2023-05-02 at 18 57 21" width="363" src="https://user-images.githubusercontent.com/107192076/235763183-5b79274d-7290-4953-8bf4-4a34c3aa150a.png">	<img alt="Screenshot 2023-05-02 at 18 58 21" width="371" src="https://user-images.githubusercontent.com/107192076/235763251-d8cd0978-2aa5-448f8-bfa7-fc01adf117f6.png">

- [ ] As suggested by the previous revieweer. This suggestion has not been attended to. Kindly to that.

### Your Design
![your-design](https://user-images.githubusercontent.com/26597432/235770504-90a3ffef-e097-4020-ac24-c3588d90e846.png)

### Figma Design
![temp-design](https://user-images.githubusercontent.com/26597432/235770541-c9ddb8f6-5233-4673-9123-bf2abab2c687.png)